File: src/main/dagger/de/thatsich/intellie/applied/aerodynamics/functionality/suite/SuiteModule.java
package de.thatsich.intellie.applied.aerodynamics.functionality.suite;

import dagger.Module;
import dagger.Provides;
import de.thatsich.intellie.applied.aerodynamics.functionality.suite.chest.AeroChest;

import javax.inject.Singleton;

/**
 @author thatsIch
 @date 24.03.2014. */
@Module(
	injects = AeroChest.class)
public class SuiteModule
{
	@Provides
	@Singleton
	AeroChest provideAeroChest ()
	{
		return new AeroChest();
	}
}

File: src/main/scala/de/thatsich/minecraft/intellie/applied/aerodynamics/intern/module/suite/SuiteModule.scala
package de.thatsich.minecraft.intellie.applied.aerodynamics.intern.module.suite


import de.thatsich.minecraft.common.log.Log
import de.thatsich.minecraft.common.module.BaseModule
import de.thatsich.minecraft.common.util.string.ID
import de.thatsich.minecraft.intellie.applied.aerodynamics.intern.module.suite.boots.HorseShoesModule


/**
 *
 *
 * @author thatsIch
 * @since 16.08.2014.
 */
class SuiteModule(modid: ID, log: Log) extends BaseModule(
	modules = Vector(
		new HorseShoesModule(modid, log)
	)
)